docs/examples/animals_template_combined.tsv

```diff
@@ -0,0 +1,5 @@
+CURIE	Label	Parent	Comment
+ID	LABEL	SC %	A rdfs:comment
+obo:0000001	animal		Any animal in the world.
+0000002	canine	animal	A member of the genus Canis.
```

Comment:
Is this combined file supposed to include this error? The file doesn't seem to be used anywhere, so I would not include it.

Reply:
To answer my own question: Yes, I think this file is included to compare row numbers for errors. So I suggest naming this file `animals_template_error.tsv` to match `animals_ext_template_error.tsv`.